src/GraphQL.Client.Abstractions.Websocket/GraphQLWebSocketRequest.cs

```diff
@@ -34,9 +34,9 @@ public string Type
         /// <summary>
         /// The payload of the websocket request
         /// </summary>
-        public GraphQLRequest Payload
+        public object Payload
         {
-            get => ContainsKey(PAYLOAD_KEY) ? (GraphQLRequest)this[PAYLOAD_KEY] : null;
+            get => ContainsKey(PAYLOAD_KEY) ? this[PAYLOAD_KEY] : null;
```

Comment:
I would prefer to change all usages of this `if (contains) then get()` pattern to `return TryGetValue(key, out v) ? v : null` to reduce dictionary key lookups.

Reply:
Also editorial change on line 17:
> The Identifier of the Response

request, not response